### DISCUSSION TITLE
Where is the history entered by the user stored?

### DISCUSSION THREAD

**Discussion by aoyamakazuya** [google/mozc#562] [Q&A]
When using Mozc for Android, there is a word displayed as "Histrory" in the lower right in Candidate word.
I think that they point to a word with input history.
I want to know where they are stored.
I 'd appreciate it if you you could answer my questions.

I apologize for my poor English.

**Comment by hiroyuki-komatsu** [comment]
Hi, I'm sorry for the too late response.

The history files are stored in [`UserProfileDirectoryImpl::GetUserProfileDirectory()`](https://github.com/google/mozc/blob/9179907357c6e8aad33a8a5e5cd39b164b2f9cc0/src/base/system_util.cc#L240).

The data are stored in multiple .db files.
Thanks,
